src/components/VoiceRecorder/VoiceRecorder.actions.js

```diff
@@ -0,0 +1,15 @@
+import { ADD_RECORD } from './VoiceRecorder.constants';
+import { START_RECORD } from './VoiceRecorder.constants';
+
+export function addRecord(newBlob) {
```

Comment:
should rename `newBlob` to just `blob` since the word "new" doesn't really add value to the meaning. the function `addRecord` doesn't care if it's new or old